runhouse/resources/hardware/cluster.py

```diff
@@ -588,18 +588,22 @@ def _is_server_up(self) -> bool:
         except ConnectionError:
             return False
 
-    def up_if_not(self):
+    def up_if_not(self, verbose: bool = True):
         """Bring up the cluster if it is not up. No-op if cluster is already up.
         This only applies to on-demand clusters, and has no effect on self-managed clusters.
 
+        Args:
+            verbose (bool, optional): Whether to stream logs from Den if the cluster is being launched. Only
+                relevant if launching via Den. (Default: `True`)
+
         Example:
             >>> rh.cluster("rh-cpu").up_if_not()
         """
         if not self.is_up():
             # Don't store stale IPs
             self.ips = []
             self.internal_ips = []
-            self.up()
+            self.up(verbose=verbose, force=False)
```

Comment:
@carolineechen does this make sense to you to have force be false here? was thinking that `up_if_not()`is a softer version of `up()`

Reply:
yea I think that makes sense, should already be false by default actually